### ISSUE TITLE
drupal init - Running drupal init in an ansible 2.7 script seems to "hang"

### ISSUE THREAD

**Issue by wonderlands-nightmare** [hechoendrupal/drupal-console#3964]
### drupal init - Running drupal init in an ansible 2.7 script seems to "hang"
**This is meant primarily as an informational item for anyone encountering a similar issue.**

### Problem/Motivation
Installing the drupal console via ansible scripts works fine using **drupal init** prior to upgrading to Ansible v2.7.

Once on Ansible v2.7 the script seems to "hang" at the **drupal init** command.

### How to reproduce
**Software versions:**
- Local OS (Ubuntu 16.04 LTS)
- VM OS (Ubuntu 14.04 LTS)
- Drupal version (Drupal 8.3.9).
- Console version (1.8.0).

**Steps to reproduce:**
1. Install ansible version between 2.0 and 2.7
2. Create ansible script tasks to install drupal using **drupal init**
3. Run ansible playbook, this should finish successfully
4. Update ansible version to 2.7
5. Run the same ansible playbook from step 3, this should stop and seem to "hang" at the **drupal init** command

### Solution
There were other developers in my team that weren't having this issue, so I attempted to roll back my VirtualBox, Composer, Vagrant, and Ansible version to match theirs but was unable to roll back Ansible as Ubuntu Package Repository didn't have the versions of Ansible my team members had.
Instead I had a team member upgrade to Ansible 2.7 and they experienced the same issue.

From there I looked in to adding a config file to "read" from for the answers but didn't find anything that would work without some larger changes.

After some trial and error and trawling through forums I found that the "issue" was that **drupal init** was actually waiting for an input like when the command is run manually.

In the end I found that using the *--site* and *--no-interaction* options for **drupal init** was the equivalent to whatever Ansible was doing to input the answers prior to v2.7.

### TL;DR
Changing **drupal init** to **drupal init --site --no-interaction** in the Ansible script "fixes" the issue.
